.github/ISSUE_TEMPLATE/new-documentation-page-suggestion.md

```diff
@@ -0,0 +1,16 @@
+---
+name: New Documentation Page Suggestion
+about: Suggest a new documentation page
+title: 'New Page: PAGE DESCRIPTION'
+labels: new_docs
```

Comment:
A page is intended to be a catch-all for any of the documentation categories so that you don't need a separate issue template to suggest a tutorial, recipe, etc. In that sense, a "page" is any of the documentation artifacts we're trying to produce.

I'm fine with not adding a default label at all since we would need to triage `new_docs` into a category anyway.